uboot-envtools: add support for gl-[ar150|domino|mifi]

kvuorine opened the pull request:
ar71xx: Change u-boot-env partition writable

Changed u-boot-env partition mount rw for:
gl-ar150
gl-domino
gl-mifi

uboot-envtools: Add ubootenv uci_config

Added ubootenv uci_config for:
gl-ar150
gl-domino
gl-mifi

Signed-off-by: Kimmo Vuorinen <kimmo.vuorinen@gmail.com>

adrianschmutzler commented on package/boot/uboot-envtools/files/ar71xx:
```diff
@@ -101,6 +101,11 @@ wndr3700)
+gl-domino|\
+gl-mifi)
+	ubootenv_add_uci_config "/dev/mtd1" "0x0" "0x8000" "0x10000"
+	;;
```
Move up to ensure alphabetic sorting.

adrianschmutzler commented:
Can you provide (and test) the same for gl-ar150 on ath79?

kvuorine commented:
> Can you provide (and test) the same for gl-ar150 on ath79?

I can, probably have some time in january.

kvuorine commented:
Somehow /etc/fw_env.config doesn't get generated on ath79 target and can't see why.

Manually creating that file using values provided will read and write ubootenv fine.

kvuorine commented:
On target ath79 board naming had changed, fixed and tested working.

adrianschmutzler commented:
Thanks. Please separate patches for ar71xx and ath79 into two separate commits.

kvuorine committed:
uboot-envtools: ar71xx: gl-domino board support

Added ubootenv uci_config for gl-ar150, gl-domino and gl-mifi

Signed-off-by: Kimmo Vuorinen <kimmo.vuorinen@gmail.com>

kvuorine committed:
uboot-envtools: ath79: gl-ar150 support

Added ubootenv uci_config for gl-ar150

Signed-off-by: Kimmo Vuorinen <kimmo.vuorinen@gmail.com>

kvuorine committed:
ar71xx: mount options to support uboot-envtools

Changed u-boot-env partition mount read-write for gl-ar150, gl-domino
and gl-mifi

Signed-off-by: Kimmo Vuorinen <kimmo.vuorinen@gmail.com>

kvuorine commented:
I had hard time making up commit messages for these changes so three commits was what I came up with.

adrianschmutzler commented:
Thanks, merged. I will backport to 19.07, too.
